stubs/google-cloud-ndb/google/cloud/ndb/_datastore_api.pyi

```diff
@@ -0,0 +1,5 @@
+from typing_extensions import Literal
+
+EVENTUAL: Literal["EVENTUAL"]
```

Comment:
I installed it from pip and this value is `2` for me, not `"EVENTUAL"`.

Reply:
Thank you very much! Will update as soon as possible.